.env.template

```diff
@@ -0,0 +1,3 @@
+TAILSCALE_API_KEY=
+TAILSCALE_EMAIL=
+TAILSCALE_NET=
```

Comment:
1. Blocking: Following a previous comment, `TAILSCALE_EMAIL` should be renamed to `TAILSCALE_ORG`.
2. Non-blocking: We should add comments here to specify the purpose of a given env var and how to obtain it.

```suggestion
# Used to automatically add provisioned VMs to your Tailscale network.
# To provision, go to https://login.tailscale.com/admin/settings/keys and hit "Generate access token..."
TAILSCALE_API_KEY=
# Used to perform operations with the Tailscale API (deleting devices, etc)
# Can be found at https://login.tailscale.com/admin/settings/general under "Organization"
TAILSCALE_ORG=
# Used to determine a provisioned VM's FQDN
# Can be found at https://login.tailscale.com/admin/dns under "Tailnet name"
TAILSCALE_NET=
```